lib/griddler/mailgun/adapter.rb

```diff
@@ -19,8 +19,8 @@ def normalize_params
           bcc: Array.wrap(param_or_header(:Bcc)),
           from: determine_sender,
           subject: params[:subject],
-          text: params['body-plain'],
-          html: params['body-html'],
+          text: params['body_plain'],
+          html: params['body_html'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.